core/src/main/scala/org/apache/spark/HeartbeatReceiver.scala

```diff
@@ -32,6 +32,8 @@ import org.apache.spark.util._
  * A heartbeat from executors to the driver. This is a shared message used by several internal
  * components to convey liveness or execution information for in-progress tasks. It will also
  * expire the hosts that have not heartbeated for more than spark.network.timeout.
+ * spark.executor.heartbeatInterval configuration value is certainly intended to be much
```

Comment:
OK, the verbatim quote from my email is a little funny with 'certainly', but, not a big deal.